data/migrations/README.md

```diff
@@ -0,0 +1,16 @@
+# Migrations for sovrin-node
+
+All migrations are python scripts with names satisfying the following rule:
+```
+<underscored_version-migration-should-be-applied-to>_<migration-name>
```

Comment:
I don't agree with the idea that "it does not cost us anything." Please collapse to 1 script and eliminate the sorting/collection. The cost of more than 1 script is complexity, as I pointed out. It's not just code complexity--it's communication/doc complexity, and complexity in testing/managing the coordination actions across multiple scripts.

Re #2: versioning needs to include a src and a target version. Of these two, the *target* is more important. We can't say, "give me a script that upgrades version X to any version later than that" -- it's just not practical. Instead, we want to say, "give me a script that upgrades X to Y". Then we can play all the migrations in order to get from X to Z.